### PR TITLE
Review branch

### PR DESCRIPTION
**Pull request by beyk**
I did:
- Create essential folders
- Add html file
- Add js files
- Install linters 
-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
```
Comment (stanleySimeon) [inline README.md:2]: - Kindly, add a professional README.md file and your project will look better!
- and update your demo link by adding `/dist`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>to do list</title>
+</head>
+
```
Comment (stanleySimeon) [inline index.html:9]: Hey @beyk  
 - You did good work, but you need to remove the blank lines in your code to make it perfect!
 
Good job, keep it up!
You're almost done!

File: index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<body>
+    <div id="container">
+        <div id="today">
+            <p id="title">Today's To Do</p>
+            <i class="fa-solid fa-arrows-rotate"></i>
+        </div>
+        <form>
+
```
Comment (stanleySimeon) [inline index.html:17]: Same as above, kindly remove the blank line!

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hey @beyk,

## Good job on your PR,

You follow all the requirements and make the project done, good job so far!

- Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.
  
  - Kindly! It can be pretty good if you create another js file for your functions and call them in index.js when needed. In that case, your code can be easier to read.
  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: README.md (hunk 1 above)
Comment (beyk) [reply README.md:2]: Thank you , I forgot it

**Pull request merged**